sass/components/_channel-header-icon.scss

```diff
@@ -0,0 +1,45 @@
+.channel-header-icon {
+    @include border-radius(4px);
+    border: none;
+    background: transparent;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    height: 32px;
+    margin: 16px 0 0 4px;
+    min-width: 32px;
+    position: relative;
+    text-align: center;
+    z-index: 5;
+    fill: v(center-channel-color-56);
+    color: v(center-channel-color-56);
+
+    &:hover {
+        background: v(center-channel-color-04);
```

Comment:
I'm not sure why we are bothering to keep the background here. It's such a light grey when using a light theme that it may as well be invisible. Show's up a bit better with a dark theme. Just my two cents so 0/5. :)

Reply:
This style is there throughout all of the buttons, so we can test it out, and decide to bump it later if we aren't satisfied with it on community.

Reply:
A more subtle design aesthetic.
Though perhaps @matthewbirtch or @andrewbrown00 or @esethna can add more to it.

Reply:
@deanwhillier thanks for the feedback, the **uxteam** took a look and agree the `hover` state at `4%` is too dim, especially on 1080p displays.

We are going to increase the `hover` state opacity to `8%`.

4% | 8%
----- | -----
![image](https://user-images.githubusercontent.com/48559658/78383934-41136600-75a7-11ea-8f81-980252b95656.png) | ![image](https://user-images.githubusercontent.com/48559658/78383904-335de080-75a7-11ea-8589-3fae943f1519.png)